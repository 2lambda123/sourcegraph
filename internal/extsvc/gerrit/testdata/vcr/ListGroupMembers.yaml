---
version: 1
interactions:
- request:
    body: ""
    form: {}
    headers: {}
    url: https://gerrit.sgdev.org/a/groups/40bb3b32f9da57a0697285daabb777965720b96e/members/
    method: GET
  response:
    body: |
      )]}'
      [
        {
          "_account_id": 1000002,
          "name": "Fake Name",
          "email": "fakename@sourcegraph.com"
        },
        {
          "_account_id": 1000018,
          "name": "Fake Name 2",
          "email": "fakename2@sourcegraph.com",
          "username": "fakename2"
        }
      ]
    headers:
      Cache-Control:
      - no-cache, no-store, max-age=0, must-revalidate
      Cf-Cache-Status:
      - DYNAMIC
      Cf-Ray:
      - 7218f418ed532943-ORD
      Content-Disposition:
      - attachment
      Content-Type:
      - application/json;charset=utf-8
      Date:
      - Sun, 26 Jun 2022 21:14:36 GMT
      Expect-Ct:
      - max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
      Expires:
      - Mon, 01 Jan 1990 00:00:00 GMT
      Pragma:
      - no-cache
      Server:
      - cloudflare
      Strict-Transport-Security:
      - max-age=15724800; includeSubDomains
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - DENY
    status: 200 OK
    code: 200
    duration: ""
