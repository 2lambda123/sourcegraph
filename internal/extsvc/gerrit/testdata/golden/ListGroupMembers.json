[
  {
   "_account_id": 1000002,
   "name": "Fake Name",
   "display_name": "",
   "email": "fakename@sourcegraph.com",
   "username": ""
  },
  {
   "_account_id": 1000018,
   "name": "Fake Name 2",
   "display_name": "",
   "email": "fakename2@sourcegraph.com",
   "username": "fakename2"
  }
 ]